test/cypress/tests/api/blocks.cy.ts

```diff
@@ -141,4 +156,179 @@ describe('api.blocks', () => {
       });
     });
   });
+
+  describe('.convert()', function () {
+    it('should convert a Block to another type if original Tool has "conversionConfig.export" and target Tool has "conversionConfig.import"', function () {
+      /**
+       * Mock of Tool with conversionConfig
+       */
+      class ConvertableTool extends ToolMock {
+        /**
+         * Specify how to import string data to this Tool
+         */
+        public static get conversionConfig(): ConversionConfig {
+          return {
+            import: 'text',
+          };
+        }
+
+        /**
+         * Specify how to display Tool in a Toolbox
+         */
+        public static get toolbox(): ToolboxConfig {
+          return {
+            icon: '',
+            title: 'Convertable tool',
+          };
+        }
+      }
+
+      const existingBlock = {
+        id: 'test-id-123',
+        type: 'paragraph',
+        data: {
+          text: 'Some text',
+        },
+      };
+
+      cy.createEditor({
+        tools: {
+          convertableTool: {
+            class: ConvertableTool,
+          },
+        },
+        data: {
+          blocks: [
+            existingBlock,
+          ],
+        },
+      }).as('editorInstance');
+
+      /**
+       * Call the 'convert' api method
+       */
+      cy.get<EditorJS>('@editorInstance')
+        .then(async (editor) => {
+          const { convert } = editor.blocks;
+
+          convert(existingBlock.id, 'convertableTool');
+        });
+
+      // eslint-disable-next-line cypress/no-unnecessary-waiting, @typescript-eslint/no-magic-numbers -- wait for block to be converted
+      cy.wait(100);
+
+      /**
+       * Check that block was converted
+       */
+      cy.get<EditorJS>('@editorInstance')
+        .then(async (editor) => {
+          const { blocks } = await editor.save();
+
+          expect(blocks.length).to.eq(1);
+          expect(blocks[0].type).to.eq('convertableTool');
+          expect(blocks[0].data.text).to.eq(existingBlock.data.text);
+        });
+    });
+
+    it('should throw an error if nonexisting Block id passed', function () {
+      cy.createEditor({}).as('editorInstance');
+
+      /**
+       * Call the 'convert' api method with nonexisting Block id
+       */
+      cy.get<EditorJS>('@editorInstance')
+        .then(async (editor) => {
+          const fakeId = 'WRNG_ID';
+          const { convert } = editor.blocks;
+
+          try {
+            convert(fakeId, 'convertableTool');
+
+            expect(true).to.be.false; // we should not pass here
+          } catch (error) {
+            expect(error.message).to.eq(`Block with id "${fakeId}" not found`);
+          }
```

Comment:
Why not use `.to.throw()`?

Reply:
Tried, but it is not working. Error fails Cypress test process.